orchestrator/graphql/resolvers/product_block.py

```diff
@@ -0,0 +1,57 @@
+from typing import Union
+
+import structlog
+
+from orchestrator.db.filters import Filter
+from orchestrator.db.filters.product_block import filter_product_blocks
+from orchestrator.db.models import ProductBlockTable
+from orchestrator.db.range.range import apply_range_to_query
+from orchestrator.db.sorting.product_block import sort_product_blocks
+from orchestrator.db.sorting.sorting import Sort
+from orchestrator.graphql.pagination import Connection, PageInfo
+from orchestrator.graphql.schemas.product_block import ProductBlock
+from orchestrator.graphql.types import CustomInfo, GraphqlFilter, GraphqlSort
+from orchestrator.graphql.utils.create_resolver_error_handler import create_resolver_error_handler
+
+logger = structlog.get_logger(__name__)
+
+
+async def resolve_product_blocks(
+    info: CustomInfo,
+    filter_by: Union[list[GraphqlFilter], None] = None,
+    sort_by: Union[list[GraphqlSort], None] = None,
+    first: int = 10,
+    after: int = 0,
+) -> Connection[ProductBlock]:
+    _error_handler = create_resolver_error_handler(info)
+
+    pydantic_filter_by: list[Filter] = [item.to_pydantic() for item in filter_by] if filter_by else []
```

Comment:
Just curious, do why do we prefix these variables with pydantic? Is that relevant?

Reply:
to not overwrite `filter_by`.